### ISSUE TITLE
Radiobutton component should render with only one label

### ISSUE THREAD

**Issue by jlevesque** [grommet/grommet#1436]
Following the comment in the PR #1359 
The radiobutton are mounting with two label thus throwing an wcag2 exception where a form element needs to have only one label. Having more then one label may confuses the screen readers.

```
<label className={classes}>
     <input {...props} className={`${CLASS_ROOT}__input`}
          type="radio" />
     <span className={`${CLASS_ROOT}__control`} />
     <label htmlFor={props.id} className={`${CLASS_ROOT}__label`}>
            {label}
      </label>
</label>
```

### Expected Behavior
The radio input should render with one label attach to it.

### Actual Behavior
The radio input currently have two label, causing accessibility issues.

### URL, screen shot, or Codepen exhibiting the issue
With the use of react-axe in my project, I was able to notice the issue. The issue is display as follow:
<img width="434" alt="screen shot 2017-07-20 at 3 44 44 pm" src="https://user-images.githubusercontent.com/6978875/28435810-92fa0ffa-6d62-11e7-86bf-d0507596f40f.png">

https://dequeuniversity.com/rules/axe/2.3/label?application=axeAPI

### Your Environment
* Grommet version: 1.5.0
* react-axe: 2.1.7

**Comment by RyanCCollins** [comment]
Hey @jlevesque !  Great catch!  It is definitely a goal of Grommet's to be accessible and we take violations of the WCAG audit seriously.  I will leave this up for now so that we can get it on our backlog.  

That said, if you feel compelled to try to help us resolve this, we would love for you to work on a PR!

**Comment by alansouzati** [comment]
fixed by https://github.com/grommet/grommet/pull/1443

Thanks again @jlevesque

**Comment by jlevesque** [comment]
Sorry for bothering in this closed issue, but I've notice a weird clicking behaviour that can be fix by changing the label tag like so : 
```
<label htmlFor={props.id} className={classes}>
    <input {...props} className={`${CLASS_ROOT}__input`}
      type="radio" />
    <span className={`${CLASS_ROOT}__control`} />
    <span className={`${CLASS_ROOT}__label`}>
        {label}
    </span>
</label>
```

**Comment by alansouzati** [comment]
can you send a PR to update that?

**Comment by jlevesque** [comment]
#1516 thanks for the quick response
